### PR TITLE
Added .editorconfig

### LINKED ISSUE #2: Missing .editorconfig

### PR DESCRIPTION
**Pull request by jkbktl**
Added .editorconfig from STRV frontend docs.

Closes #2

### CODE REVIEW

**Comment by dannytce** [comment]
@jkbktl please also add it to `lib/dotfiles/.editorconfig` and call `copyDotfile` in `lib/configs/default.js`

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# editorconfig.org
+
+# This is the project's root directory
```
Comment (prichodko) [inline .editorconfig]: Please remove comments, it's cleaner and nobody looks or modifies this file anyway.

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+root = true
+
+[{*, .*}]
```
Comment (prichodko) [inline .editorconfig]: Just use `[*]`.

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+# Use Unix line endings
+end_of_line = lf
+# The files are utf-8 encoded
+charset = utf-8
+# No whitespace at the end of line
+trim_trailing_whitespace = true
+# A file must end with an empty line - this is good for version control systems
+insert_final_newline = true
+# A line should not have more than this amount of chars (not supported by all plugins)
+max_line_length = 100
```
Comment (prichodko) [inline .editorconfig]: You can safely remove this one, because line length is handled by `Prettier` formatting.

File: .editorconfig [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+[{Makefile,**.mk}]
```
Comment (prichodko) [inline .editorconfig]: You can delete this configuration, we don't use makefiles.

**Comment by prichodko** [comment]
Although I am still kinda skeptical about this and personally won't ever use it, I understand it could be useful on projects where potentially many people collaborate.

**Pull request merged**